docs/prover/changelog/gui_changelog.md

```diff
@@ -4,6 +4,17 @@ GUI Release Notes
 ```{contents}
 ```
 
+5.0.1 (May 18, 2025)
+---------------------------
+
+### Features
+
+- [feat] Rule Report - Nesting In Variables Tab: Enhance the Variables tab by introducing aggregation functionality to env and method variables. This will simplify scrolling, navigation, and comprehension for users by mirroring the aggregation behavior in the Call Trace for complex storage types (such as structs, arrays, and maps).
```

Comment:
```suggestion
- [feat] Rule Report - Nesting In Variables Tab: Enhance the Variables tab by introducing aggregation functionality to `env` and method variables. This will simplify scrolling, navigation, and comprehension for users by mirroring the aggregation behavior in the Call Trace for complex storage types (such as structs, arrays, and maps).
```